### ISSUE TITLE
Cannot find module 'pseudomap'

### ISSUE THREAD

**Issue by rbustami** [isaacs/node-lru-cache#114]
I am unable to require the the LRU cache module. I've run npm install lru-cache --save per the instructions. For some reason the pseudomap module used by lru-cache cannot be found even though it is present in my node_modues. I even tried installing pseudomap separately. 

Here is my code:
```
const LRU = require(`lru-cache`);
```

Here is the error:
```
Error: Cannot find module 'pseudomap'
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.Module._load (module.js:386:25)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/myProject/node_modules/lru-cache/index.js:7:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
```

I'm using node v6.1.0 and npm 4.2.0.

**Comment by akiva** [comment]
+1 I, too, am experiencing this with a recent project I am working on. My guess is that it's caused by it's inclusion in `bower`. The odd thing is, this package itself seems to have no dependency on `pseudomap` directly. Investigating.

**Comment by akiva** [comment]
OK, solved. 

This was an error caused in `bower-registry-client`. I was going to fix the bug and do a PR, but I noticed the `package.json` points to a now missing github repo (_shame on you, bower), thus leaves a broken module in the npm user-space. Looking further, I see that they merged the `bower-registry-client` into the mainline `bower` repo on June 10, 2016, resolving the issue (which was caused by the poor choice of using a `~` instead of a `^` for it's `lru-cache` dependency, as it had since introduced `pseudomap` internally).

In summary, this can be resolved by insuring you are declaring a dependency on `bower` internally in your own `package.json` file using `^` and not `~`. This fixed the problem for me. I have no love for `bower`, but everyone makes mistakes. It can be especially easy without care when using `semver`.

**Comment by asafch-talenya** [comment]
@akiva , might you be able to give me a hand?
I'm experiencing the exact same issue, but my project doesn't use `bower` at all. I'm experiencing this as part of a pre-commit script that runs `eslint`.

Stack trace:
```
Error: Cannot find module 'pseudomap'
			    at Function.Module._resolveFilename (module.js:485:15)
			    at Function.Module._load (module.js:437:25)
			    at Module.require (module.js:513:17)
			    at require (internal/module.js:11:18)
			    at Object.<anonymous> (/Users/asaf/work/backend/node_modules/cross-spawn/node_modules/lru-cache/lib/lru-cache.js:5:11)
			    at Module._compile (module.js:569:30)
			    at Object.Module._extensions..js (module.js:580:10)
			    at Module.load (module.js:503:32)
			    at tryModuleLoad (module.js:466:12)
			    at Function.Module._load (module.js:458:3)
```

Node v. 8.1.4
NPM v. 5.0.3

**Comment by akiva** [comment]
@asafch-talenya Glad to hear you got it all sorted out. It can catch you off guard, forgetting to delete `package-lock.json`. :)

@isaacs I feel this ticket can be closed. Any objections, @rbustami ?
